doc/install.rst

```diff
@@ -50,39 +53,38 @@ Ginga can draw its output to a number of different back ends.
 Depending on which GUI toolkit you prefer (and what you want to
 do), you will need at least one of the following:
 
-* python-qt4
-* python-qt5
-* python-pyside (qt4/qt5 alternative)
-* python-gtk (gtk2) **AND** python-cairo
-* python gtk3 (gi) **AND** python-cairo
-* python-Tkinter
+.. TODO: This can be broken down in a clearer way.
+* QtPy (PyQt4 or PyQt5)
+* PySide (Qt4/Qt5 alternative)
+* PyGTK (gtk) **AND** `pycairo <https://github.com/pygobject/pycairo>`_ (GTK 2)
+* pygobject (gi) **AND** pycairo (GTK 3)
+* `tkinter <https://docs.python.org/3/library/tk.html>`_
 * matplotlib
 * tornado
-* aggdraw
-* PIL (pillow)
-* OpenCv
+* `aggdraw <https://github.com/pytroll/aggdraw>`_
+* Pillow (PIL fork)
 
 RECOMMENDED
 ===========
+
 Certain plugins in the reference viewer (or features of those plugins)
 will not work without the following packages:
 
 * matplotlib (required by: Pick, Cuts, Histogram, LineProfile)
-* webkit (required by: WBrowser (used for online help))
```

Comment:
And only for online help functionality...  I think with `conda` webkit comes bundled with Qt, but not in other distributions necessarily.

Reply:
But what is it called in PyPI?

Reply:
Looks like for Python 3, `pygobject` includes `webkit`.  For gtk2, it may be [pywebkitgtk](https://pypi.org/project/pywebkitgtk/). But I think you don't need to worry about gtk2 right now.  We could just have a `pip-requirements-gtk3` file.

I'm thinking of deprecating gtk2 in the not too distant future.  Probably after Ginga 3.0 though.

Reply:
Okay, I renamed the file to be obvious that it only applies to `gtk3` in a follow-up commit.